ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -4,12 +4,12 @@ let prototyped_of_class = function
 
 let prototyped_of_field = function
   | "Repository", "gpgkey_path" ->
-      Some "22.10.0-next"
+      Some "22.12.0"
```

Comment:
Maye this should not be in a PR and we deal with this once it is merged?

Reply:
I can remove this before merging, I committed because it otherwise disrupts the work-flow as the change reappears every time the project is compiled